.github/workflows/pr.yaml

```diff
@@ -54,13 +54,13 @@ jobs:
         shell: bash
   libbpfgo-unit-tests:
     name: libbpfgo Unit Tests
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-24.04
     strategy:
       matrix:
-        go-version: [ 1.18, 1.19, '1.20', '1.21', 'stable' ]
+        go-version: [ '1.21', '1.22', 'stable' ]
```

Comment:
@rscampos as we're using toolchain now, it's available from 1.21, that's why I removed the older ones.